src/styles/mixins/_typography.scss

```diff
@@ -42,3 +42,30 @@
     line-height: $bdl-line-height;
     letter-spacing: .3px;
 }
+
+@mixin target-ie11 {
+    @media all and (-ms-high-contrast: none), (-ms-high-contrast: active) {
+        @content;
+    }
+}
+
+@mixin line-clamp($max-lines: 2, $line-height: 1.5) {
+    display: -webkit-box; /* stylelint-disable-line value-no-vendor-prefix */
+    overflow: hidden;
+    line-height: $line-height;
+    white-space: normal;
+    overflow-wrap: break-word;
+
+    /* autoprefixer: ignore next */
```

Comment:
is this needed? it looks like it might've been fixed https://github.com/postcss/autoprefixer/issues/1220

Reply:
My understanding is it is still needed because we introduced the EUA version last year, which was after this fix.

Reply:
ah I see. the fix was released in 9.6 and EUA is at 10 so I'm not sure if it was originally needed